### DISCUSSION TITLE
How to override user permission from role for certain users?

### DISCUSSION THREAD

**Discussion by francissebintopscore** [spatie/laravel-permission#1915] [Q&A]
Hi, 
Thanks in advance, 

How to override user permission to certain users?. The scenario is the following thing.
There is a role called editor, the editor has the privilege to create, edit, view, and delete posts. I need certain users with the role editor to revoke permission to delete posts. 

User A (Editor role)=> can create, edit, view, and delete posts
User B (Editor role)=> can create, edit, view, and there is no delete option for posts

How can I achieve this with the Spatie package or any recommendation of a package that is able to do so?

**Comment by erikn69** [comment]
You can achieve that reading older answered discussions instead of open a new one 😂
